### DISCUSSION TITLE
Reworking documentation & dropping FR support

### DISCUSSION THREAD

**Discussion by Mara-Li** [ObsidianPublisher/obsidian-github-publisher#86] [Announcements]
Hello!
I reworked, again, the [Documentation](https://obsidian-publisher.netlify.app/).

Notably : 
- Correct grammar and syntax error, mainly using Chat-GPT because it's too much for my little french brain
- Removed the French translation : **I can't maintain, the French translation in solo. It's too many works, as I need to duplicate each file.*

Moreover, regarding the ex-discord server : I deleted the server. Why? Because I can't help everyone in private. If you need help, please, ask here. It will help future people, as you can search conversation and issue. 

Also, I won't provide help in DM. I just, can't, help everyone. 

On this project, i'm literally **one**, and English is not my primary language. 

I know my autistic ass made a maybe difficult project to maintain. That's why I take the whole day to rewrite the entire documentation using the code, the old docs and Chat-GPT to correct my grammar error. 

Occasionally, I have taken the entire night to check the bug and strange error some user have. For now, I'm jobless, but I future, I just can't continue like that. For an example, I'm on this since the 9 a.m and I just finished, at 11 pm. 

So, please, if you need help: 
- Create a discussion on the forum, following the template. 
- Or create an issue 

If you create an issue, I BEG YOU to make a reproducible bug. Aka : 
- Data.json
- A reproducible vault with one or two example file ; 
- A link to your destination repository.

Ask the good question. Check the documentation. Don't want too much on a plugin. 

---

I hope, I really hope, that my work will help you. I'm a little sad to see I don't have much download (probably because my README was unreadable ?).

If I took time to help you, don't hesitate to send me some kofi : https://ko-fi.com/X8X54ZYAV

**Comment by smaroukis** [comment]
I think this project is great! I was making some custom python scripts to export to my jekyll site, but I'm going to play around with yours instead. I'll try to help with the docs as well. I think this can plug in well into multiple ecosystems.

**Reply by Mara-Li** [reply]
thanks !

**Comment by andrey-jef** [comment]
Wow. How can a gem like you, who developed such useful plugin and documentation tool be currently jobless? Is it really competitive in France's software engineering job market? I wish that you meet an employer who evaluate highly your talent.

**Reply by Mara-Li** [reply]
In fact, I don't study software engineering (yeah, I'm a hobbyist!) but epidemiology and health risk management...
